### ISSUE TITLE
Add ability to wrap contents of table.

### ISSUE THREAD

**Issue by Robpol86** [Robpol86/terminaltables#5]
Provide API in terminaltables for textwrap instead of having developers do it themselves.

**Comment by d33tah** [comment]
I second that. It would be perfect if it could also figure out the maximum length for a column, depending on terminal width and amount of space left.

**Comment by d33tah** [comment]
Here's what I use right now: https://github.com/d33tah/dotfiles/commit/f7a81f587e345debf2c262e6838e021256fd2b74

**Comment by rmerkushin** [comment]
yeah, it would be nice if it could also split table on many tables for very long tables like this:
[![Снимок экрана 2016-12-20 в 18.36.34.png](https://s28.postimg.org/semk9uphp/2016_12_20_18_36_34.png)](https://postimg.org/image/qa278rnux/)
